shopinvader_api_address/routers/address_service.py

```diff
@@ -0,0 +1,147 @@
+from typing import Annotated, List
+
+from fastapi import APIRouter, Depends
+
+from odoo.addons.base.models.res_partner import Partner as ResPartner
+from odoo.addons.fastapi.dependencies import authenticated_partner
+from odoo.addons.shopinvader_schema_address.schemas import (
+    BillingAddress,
+    ShippingAddress,
+)
+
+from ..schemas import (
+    BillingAddressCreate,
+    BillingAddressUpdate,
+    ShippingAddressCreate,
+    ShippingAddressUpdate,
+)
+
+# create a router
+address_router = APIRouter(tags=["addresses"])
+
+# --- Billing addresses ---
+
+
+@address_router.get("/addresses/billing", response_model=List[BillingAddress])
+@address_router.get("/addresses/billing/{address_id}", response_model=BillingAddress)
```

Comment:
AFAIK the `response_model` is no more required and is deduced from the return type of the decorated method. In this specific case the problem will be that the return type depends on the route used to call the method. We should maybe have 1 method by route in this case.

Reply:
Indeed, I have split the get function in 2